web-common/src/features/explores/selectors.ts

```diff
@@ -239,7 +237,12 @@ export function shouldRedirectToViewWithParams(
     defaultExplorePreset,
   );
   // copy over any partial params. this will include the view and measure param
-  url.searchParams.forEach((value, key) => newUrl.searchParams.set(key, value));
+  url.searchParams.forEach((value, key) => {
+    // ignore `view` param since it is already part of partialExploreState
+    // in case `view` was same as default adding it here would mess up the order.
+    if (key === ExploreStateURLParams.WebView) return;
+    newUrl.searchParams.set(key, value);
+  });
```

Comment:
It looks like this logic should go in the `convertExploreStateToURLSearchParams()` function. Whenever the `view` is the default view, it shouldn't be stringified into query params (which it looks like the `shouldSetParam()` function is for).

Reply:
Good point. But `convertExploreStateToURLSearchParams` is not the one adding the params here. It is `getUrlForWebView` that always set the view, it should not set it if it is default.

Reply:
I think it's fine for it to set explicit parameters, then the `load` function can remove (via redirect) any parameters that are defaults